tools/cicd/cloudbuild.yaml

```diff
@@ -32,7 +32,6 @@ steps:
 - name: us-docker.pkg.dev/$PROJECT_ID/tools/hatch:${_PYTHON_VERSION}
   id: tests
   args:
-  - run
```

Comment:
@rosbo When I changed this line, it caused failures of cicd with error that "test" is not supported by hatch. I wonder if our `Google Cloud Build / kagglehub-branch-3-12-0 (kaggle-cicd)` is using a different docker image other than the Dockerfile defined here? How should I debug this?

Reply:
Yes, you will need to update the `cicd` Dockerfile: https://github.com/Kaggle/kagglehub/blob/main/tools/cicd/Dockerfile